src/get-started/quickstart.md

```diff
@@ -74,8 +88,9 @@ $ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
 ```
 
 {:.runner-flink}
-``` 
-TODO BEAM-899
+```
+$ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
```

Comment:
I was thinking of putting that in the Flink Runner-specific page to not overload this because it requires some more involved steps.

If you want, I can also put that here, though.